apps/web/components/CheckoutPayment/types.ts

```diff
@@ -0,0 +1,15 @@
+export enum PaymentMethods {
```

Comment:
Suggestion:
In this project as well as in SFUI we were naming enums using singular form, so:
```suggestion
export enum PaymentMethod {
```

It might be better to stick with that principle, what do you think?

Reply:
Aye! Changed 673544dfc5d86accf365515244844a44541d4669